pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint

```diff
@@ -0,0 +1,20 @@
+"Blocked TCP response" issues:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In the connection from test-vpc1-ky/db-endpoint-gateway-ky[10.240.30.6] to test-vpc1-ky/vsi1-ky[10.240.10.4] TCP src-ports: 1-99,201-65535; TCP src-ports: 100-200 dst-ports: 1-9,51-65535 response is blocked
```

Comment:
consider improve wording in a follow up PR

Reply:
I'll ask @kyorav to help us with the wording and @nevo to take another look at the `connectionLinter` interface